### PR TITLE
Write a better `README.md` file

### PR DESCRIPTION
**Pull request by Hywan**
We need a “Why?” section and more information about authors, roadmap, milestones, board etc.

### CODE REVIEW

**Comment by Hywan** [comment]
I am waiting for a peer-review from @krtek4!

File: README.md [hunk 1]
```diff
@@ -1,14 +1,54 @@
 # The A11y Machine
 
-**The A11y Machine** is an **automated accessibility testing tool** which
-**crawls** and **tests** all pages of any web application. It validates pages
-against the following specifications/laws:
+**The A11y Machine** (or `a11ym` for short, spelled “alym”) is an **automated
+accessibility testing tool** which **crawls** and **tests** all pages of any web
```
Comment (krtek4) [inline README.md]: -all since it depends on the parameters, it bring nothing to the description

File: README.md [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
+application, to finally produce detailed and efficient reports. It validates
```
Comment (krtek4) [inline README.md]: 'finally' is not needed IHMO, it only make the sentence "heavier". Not sure about the 'efficient' also, I don't like bragging ^^ and what is an 'efficient report' anyway ?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,54 @@
...
   * U.S. [Section 508](http://www.section508.gov/) legislation.
 
+## Table of contents
+
+* [Why?](#why)
+* [Installation](#installation)
+* [Usage](#usage)
+  * [Possible output](#possible-output)
+  * [How does it work?](#how-does-it-work)
+  * [Write your custom sniffers](#write-your-custom-sniffers)
```
Comment (krtek4) [inline README.md]: the word "sniffer" was not explained at this time, maybe "rules" ?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,54 @@
...
+* [Roadmap and board](#roadmap-and-board)
+* [Authors and license](#authors-and-license)
+
+## Why?
+
+If **privacy** matters for you, you're likely to install The A11y Machine over
```
Comment (krtek4) [inline README.md]: unlikely ?

Otherwise I don't get your meaning

File: README.md [hunk 5]
```diff
@@ -1,14 +1,54 @@
...
+any SaaS services: It runs locally so you don't need to send your code
```
Comment (krtek4) [inline README.md]: is HTML code ? just :trollface:

File: README.md [hunk 6]
```diff
@@ -1,14 +1,54 @@
...
+somewhere, you can test all parts of your application including the ones which
+requires an authentification (like a checkout, a back-office etc.)…
+
+Here is the pros and cons of The A11y Machine vs. other third-party/SaaS
+services put in a table:
```
Comment (krtek4) [inline README.md]: This sentence seems unnecessarily long :

"Here are some pros and cons compared to other solutions:"

File: README.md [hunk 7]
```diff
@@ -1,14 +1,54 @@
...
+
+Properties | The A11y Machine | SaaS services
+-----------|------------------|---------------
+Can run locally              | yes | no
+Can test each patch          | yes | no (except if commit)
```
Comment (krtek4) [inline README.md]: deployed instead of commit ?

File: README.md [hunk 8]
```diff
@@ -1,14 +1,54 @@
...
+Reduce the test loop         | yes | no (the loop is longer)
+Can test private code        | yes | no (you must send your code)
+Can test auth-required parts | yes | no
+Can crawl all your pages     | yes | no (it depends of your money)
```
Comment (krtek4) [inline README.md]: "on your money"...

Probably more right : yes, but it can be pricey

File: README.md [hunk 9]
```diff
@@ -1,14 +1,54 @@
...
+
+Accessibility is not only a concern for disabled people: The biggest disabled
+peoples are bots, like [DuckDuckGo](https://duckduckgo.com),
```
Comment (krtek4) [inline README.md]: not sure "bots" are "people".

Maybe : "bots can also be considered as such" (which doesn't really remove the people comparison, but attenuate it a bit)

File: README.md [hunk 10]
```diff
@@ -1,14 +1,54 @@
...
+[Google](https://google.com/) or [Bing](https://bing.com/). By respecting these
+standards, you're likely to have a better referencing. Also it helps to clean
```
Comment (krtek4) [inline README.md]: Not sure "referencing" is the right word, ranking ? visibility ?

File: README.md [hunk 11]
```diff
@@ -1,14 +1,54 @@
...
+your code. Accessibility might often not be addressed because of its cost, but
+what cost the most is finding errors! With The A11y Machine, this cost is
+aggressively reduced and you can focus on fixing reported errors. We heard about
+stories fixing thousands of errors in a day.
```
Comment (krtek4) [inline README.md]: cost / cost / cost ... last part is a lot of bragging also ;)

"Accessibility issues are often left unaddressed for budget reasons. In fact most of the cost is spent looking for errors on your website. The A11y Machine greatly help with this task, you can thus focus on fixing your code and reap the benefits."

wdyt ?

File: README.md [hunk 12]
```diff
@@ -20,7 +60,7 @@ $ npm install the-a11y-machine
 
 ## Usage
 
-First, see the help:
+As a prelude, see the help:
```
Comment (krtek4) [inline README.md]: Can I call you Poetic Lover ? :P

**Comment by krtek4** [comment]
Finished commenting on the proposed change. The rest of the README might also need some reviewing, but let's focus on features for now ;)

File: README.md (hunk 1 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 3 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 4 above)
Comment (Hywan) [inline README.md]: If you are concerned by privacy, then a11ym is for you.

File: README.md (hunk 5 above)
Comment (Hywan) [inline README.md]: Not only the HTML, but the text. The text can be private :-].

File: README.md (hunk 6 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 7 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 8 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 9 above)
Comment (Hywan) [inline README.md]: :+1:

File: README.md (hunk 10 above)
Comment (Hywan) [inline README.md]: Ranking!

File: README.md (hunk 11 above)
Comment (Hywan) [inline README.md]: I think I hate acronyms. But :+1: ;-).

File: README.md (hunk 12 above)
Comment (Hywan) [inline README.md]: Sure!

File: README.md (hunk 4 above)
Comment (krtek4) [inline README.md]: Yeah, but that's not how your sentence read for me :

"If privacy matters you are likely to install a11ym on a SaaS service."

I think this should be reworded

File: README.md (hunk 5 above)
Comment (krtek4) [inline README.md]: Text is not code either. "files" ?

**Pull request merged**